### ISSUE TITLE
xlsx-0.7.0 test suite failure with conduit-1.3

### ISSUE THREAD

**Issue by DanBurton** [qrilka/xlsx#113]
As seen on the Stackage build server:

```
...

  write . fast-read == id:         FAIL
    Exception: InvalidFile "xl/worksheets/sheet1.xml" "Bad entity \"&#34;\""
    CallStack (from HasCallStack):
      error, called at src/Codec/Xlsx/Parser.hs:86:22 in xlsx-0.7.0-HMONWxsa71rI
tUFOdhVMUd:Codec.Xlsx.Parser
  fromRows . toRows == id:

...

1 out of 28 tests failed (5.82s)
```

I was able to reproduce locally like so:

```bash
stack unpack xlsx-0.7.0 && cd xlsx-0.7.0
edit stack.yaml # add the following stack.yaml
stack test
```

```yaml
# stack.yaml
resolver: nightly-2018-03-10
extra-deps:
- conduit-1.3.0
- conduit-extra-1.3.0
- resourcet-1.2.0
- xml-conduit-1.8.0
```

It is unclear to me whether this is a testing fluke, or a legitimate regression in xlsx or one if its dependencies (xml-conduit, perhaps?). For now, I will mark xlsx as an expected test failure on Stackage.

**Comment by DanBurton** [comment]
Sorry, edited the title. This is a test suite failure, not a build failure.

**Comment by qrilka** [comment]
Thanks @DanBurton for letting me know, I'll look into it

**Comment by qrilka** [comment]
Fixed in 71be3e85c4ae3f1cd4d6c21778138cd82be401ad and released as 0.7.1
